src/telemetry.h

```diff
@@ -218,9 +218,9 @@ struct Telemetry {
 };
 
 enum {
-    PROTO_TELEM_UNSUPPORTED = -1,
-    PROTO_TELEM_OFF = 0,
-    PROTO_TELEM_ON  = 1,
+    PROTO_TELEM_UNSUPPORTED = 0,
+    PROTO_TELEM_OFF = 1,
+    PROTO_TELEM_ON  = 2,
 };
 
 
```

Comment:
I haven't looked at it in detail, but will this break existing model files?

Reply:
It doesn't seem to be model file related. It is only used to activate telemetry alarms and in pages

Reply:
Shouldn't break models.  Looks like all the protocols use local defines for the protocol option values.